webpack-sls-offline.config.js

```diff
@@ -3,7 +3,7 @@ const slsw = require('serverless-webpack');
 
 module.exports = {
   target: 'node',
-  mode: 'production',
+  mode: 'development',
```

Comment:
What affect does this change have?

Reply:
unable to connect to a local database using mysql2 and mysql5.7 without this. i believe it only effects local running and not when deployed.